src/components/AttachmentModal.js

```diff
@@ -148,7 +165,14 @@ class AttachmentModal extends PureComponent {
                         shouldShowDownloadButton={!this.props.isUploadingAttachment}
                         onDownloadButtonPress={() => fileDownload(sourceURL)}
                         onCloseButtonPress={() => this.setState({isModalOpen: false})}
-                        subtitle={this.props.originalFileName ? this.props.originalFileName : lodashGet(this.state, 'file.name', '')}
+                        subtitle={(
+                            <TextWithEllipsis
+                                leadingText={fileName.trim()}
+                                trailingText={fileExtension ? `.${fileExtension.trim()}` : ''}
```

Comment:
```suggestion
                                trailingText={fileExtension ? `.${fileExtension}` : ''}
```

we don't need to trim here if we do this above.